diff --git a/node_modules/@radix-ui/react-collection/dist/index.mjs b/node_modules/@radix-ui/react-collection/dist/index.mjs
index b33b499..e78345c 100644
--- a/node_modules/@radix-ui/react-collection/dist/index.mjs
+++ b/node_modules/@radix-ui/react-collection/dist/index.mjs
@@ -73,7 +73,7 @@ var import_react2 = __toESM(require("react"));
 // src/Collection.tsx
 var import_react_context = require("@radix-ui/react-context");
 var import_react_compose_refs = require("@radix-ui/react-compose-refs");
-var import_react_slot = require("@radix-ui/react-slot");
+var import_react_slot = require("@radix-ui/react-slot");
 var import_react3 = __toESM(require("react"));
 var Collection = ({
   slot,
@@ -107,7 +107,7 @@ var Collection = ({
   const composedRefs = (0, import_react_compose_refs.useComposedRefs)(ref, itemRef);
   return /* @__PURE__ */ (0, import_react3.createElement)(import_react_context.Slottable, null, /* @__PURE__ */ (0, import_react3.createElement)(
     "div",
-    {
+    {...rest,
       ref: composedRefs,
       ...itemData
     },
@@ -155,4 +155,4 @@ var createCollection = (name) => [
   return getItems;
 };
 
-export { Collection, createCollection };
+export { Collection, createCollection, import_react_slot as createSlot };
 //# sourceMappingURL=index.mjs.map
